client-runtime/src/main/java/com/microsoft/rest/v2/RestProxy.java

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+
+package com.microsoft.rest.v2;
+
+import com.microsoft.rest.RestClient;
+import com.microsoft.rest.v2.annotations.GET;
+import com.microsoft.rest.v2.annotations.Host;
+import sun.reflect.generics.reflectiveObjects.NotImplementedException;
+
+import java.lang.reflect.InvocationHandler;
+import java.lang.reflect.Method;
+import java.lang.reflect.Proxy;
+
+public class RestProxy implements InvocationHandler {
+    private String host;
+    private RestClient restClient;
+
+    private RestProxy(String host, RestClient restClient) {
+        this.host = host;
+        this.restClient = restClient;
+    }
+
+    @Override
+    public Object invoke(Object proxy, Method method, Object[] args) throws Throwable {
+        if (method.isAnnotationPresent(GET.class)) {
+            String relativePath = method.getAnnotation(GET.class).value();
+            return (host + "/" + relativePath).getBytes();
```

Comment:
Is this supposed to actually make the REST API network call at this point using RxNetty/OkHttp? I'm confused why the proxy is just returned the bytes of the GET URL.